ui/src/Live/Multiview.tsx

```diff
@@ -106,18 +109,62 @@ function selectedReducer(old: SelectedCameras, op: SelectOp): SelectedCameras {
  */
 const Multiview = (props: MultiviewProps) => {
   const [searchParams, setSearchParams] = useSearchParams();
-  const theme = useTheme();
 
   const [selected, updateSelected] = useReducer(
     selectedReducer,
     searchParams.has("cams")
-      ? JSON.parse(searchParams.get("cams") || "")
+      ? JSON.parse(searchParams.get("cams") || "") :
+      localStorage.getItem("camsSelected") !== null ?
```

Comment:
So this honors the URL parameters if set, and then tries local storage otherwise? Neat!

Reply:
Yes!